apps/okreads/browser/src/app/app.component.html

```diff
@@ -3,7 +3,7 @@
     <header class="header">
       <div class="header--content">
         <h1>okreads</h1>
-        <div>
+        <div role="button" tabindex="0" aria-pressed="false">
```

Comment:
We actually don't need these attributes, since there's a button inside of this div that has all of this functionality.